src/ng-select/lib/ng-select.component.spec.ts

```diff
@@ -3560,13 +3560,13 @@ class NgSelectTestCmp {
         this.visible = !this.visible;
     }
 
-    onChange(_: Event) {
+    onChange(_: any) {
```

Comment:
I would prefer not changing types to any if we can be more specific

Reply:
In this case it's the angular `$event` so I think it's more descriptive as `any`. Also `ngc` fails as it's not used as `Event`